src/Assets/Fonts.tsx

```diff
@@ -5,6 +5,10 @@ const GaramondSizes = {
     size: "11px",
     height: "1.1em",
   },
+  s14: {
+    size: "14px",
+    height: "18px",
```

Comment:
Noticed that the new text options weren't using the new line heights in [https://www.notion.so/artsy/Typography-d1f9f6731f3d47c78003d6d016c30221](here) -- is that planned as part of the responsive work?

Reply:
Yes, I was pairing on the responsive type system work with @yuki24 and @erikdstock yesterday to get things moving. https://github.com/artsy/palette/blob/initial-setup/src/Text.tsx. We'll switch to that soon after we figure out some details.